opm/simulators/linalg/cuistl/SolverAdapter.hpp

```diff
@@ -33,7 +33,11 @@
 #include <opm/simulators/linalg/cuistl/PreconditionerAdapter.hpp>
 #include <opm/simulators/linalg/cuistl/detail/has_function.hpp>
 
-
+#ifdef OPEN_MPI
+#if OPEN_MPI
+#include "mpi-ext.h"
+#endif
+#endif
```

Comment:
I am a bit surprised why it is done this way (everything tailored towards OpenMPI). It seems like many MPI implementations support this.

Is there a special reason why we focus only on one?